execute/tokendata/usdc/attestation.go

```diff
@@ -54,48 +66,49 @@ type AttestationEncoder func(context.Context, cciptypes.Bytes, cciptypes.Bytes)
 //	12 ->
 //	  0 -> AttestationStatus{Error: nil, Attestation: "ABCDEF", MessageHash: bytes}
 //	  2 -> AttestationStatus{Error: ErrNotRead, Attestation: nil, MessageHash: nil}
-type AttestationClient interface {
-	Attestations(
-		ctx context.Context,
-		msgs map[cciptypes.ChainSelector]map[reader.MessageTokenID]cciptypes.Bytes,
-	) (map[cciptypes.ChainSelector]map[reader.MessageTokenID]AttestationStatus, error)
-}
-
-type sequentialAttestationClient struct {
+//type AttestationClient interface {
```

Comment:
Why comment out this interface? If not needed please remove, don't leave commented out code unless there is a good reason to do so

Reply:
This interface was introduced to allow smarter processing for Attestations APIs which allow batching. I would expect that Lombard implementation would reimplement that and get all attestations within a single call, no?

Reply:
Ahh, it's moved to a different file. Please remove it from here and make sure godoc is also moved with the code